recipes/versiontools/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{%set name = "versiontools" %}
+{%set version = "1.9.1" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "a969332887a18a9c98b0df0ea4d4ca75972f24ca94f06fb87d591377e83414f6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  preserve_egg_dir: True
+
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    # - setuptools
+
+test:
+  imports:
+    - versiontools
+    - versiontools.setuptools_hooks
+    - versiontools.bzr_support
+    - versiontools.git_support
+    - versiontools.hg_support
+
+about:
+  home: https://launchpad.net/versiontools
+  license: LGPL 3.0
+  # license_file: No LICENSE included in package
+  license_family: Public-Domain
```

Comment:
...or perhaps GPL then? Not sure how the license families work. @goanpeca, could you please weight in here?